README.md

```diff
@@ -26,7 +26,7 @@ And configure the library, the possible configurations are as follow:
 - `projectId`: The ID of you project, you get it from your project interface after your register to sections
 - `environment`: _to use only for development purposes_ set it to "testing" if you want your requests to be directed to sections test server
 - `projectUrl`: _to use only if you intend to run sections in SSR_ set it to the project url you defined in your project interface on sections back office.
-- `queryStringSupport`: _to use only if you intend to use query strings on your project_ set it to `enabled`. Enabling it on a project that does not have access to query strings will return errors when getting pages. (query string example: `/blogs1/blogs2/path=/blogs/article_path/categories_titles=Science,People` where pagePath is `/blogs1/blogs2` and query strings are `path1=/blogs/article_path` and `categories_titles=Science,People`)
+- `queryStringSupport`: _to use only if you intend to use query strings on your project_ set it to `enabled`. Enabling it on a project that does not have access to query strings will return errors when getting pages. (query string example: `/blogs1/blogs2/path=/blogs/article_path/categories_titles[]=Science,People` where pagePath is `/blogs1/blogs2` and query strings are `path1=/blogs/article_path` and `categories_titles[]=Science,People`). It is important to add `[]` after the QS key if the value of the QS should be an array like `categories_titles[]=Science,People`. You can as well use regular query strings as follow: `/blogs1/blogs2?path=/blogs/article_path&categories_titles[]=Science,People`. Note that you can not use both ways together
```

Comment:
I don't see the difference between your two examples in the above sentence ?
When the QS is an array or a regular one you are using the same notation for categories_titles[]

Reply:
What I meant by regular query string is the QS that starts with `?` so `?categories_titles[]=` instead of `/categories_titles[]=`. I updated the sentence to be more clearer